lib/src/trailer.rs

```diff
@@ -40,6 +40,12 @@ pub struct Trailer {
     pub value: TrailerValue,
 }
 
+impl Into<(TrailerKey, TrailerValue)> for Trailer {
```

Comment:
I guess this is automatically implemented with the `From` implementation. @TheNeikos?